Android/wallet/src/main/java/com/flow/wallet/wallet/BaseWallet.kt

```diff
@@ -33,6 +33,7 @@ interface Wallet {
     suspend fun refreshAccounts()
     suspend fun fetchAccounts()
     suspend fun fetchAccountsForNetwork(network: ChainId): List<FlowAccount>
+    suspend fun awaitFirstAccount()
```

Comment:
can we make the wallet or accounts as livedata, so the wallet client just subscript to this value, instead of using await func.